classes/Media.php

```diff
@@ -255,11 +255,11 @@ public static function getMediaPath($media_uri, $css_media_type = null)
 		// check if css files exists
 		if (!array_key_exists('host', $url_data))
 		{
-			if (!@filemtime($file_uri) || @filesize($file_uri) === 0)
+			if (!is_file($file_uri))
```

Comment:
Why? You are just checking for true/false not for the real file size.

`@filesize($file_uri) === 0`

an empty file would be true, like with `is_file`